man/systemd-sysext.xml

```diff
@@ -33,15 +33,28 @@
       <arg choice="plain">COMMAND</arg>
     </cmdsynopsis>
 
-    <para><filename>systemd-sysext.service</filename></para>
+    <para><simplelist>
+      <member><filename>systemd-sysext.service</filename><member>
+      <member><filename>/etc/systemd/sysext.conf</filename></member>
+      <member><filename>/run/systemd/sysext.conf</filename></member>
+      <member><filename>/usr/local/lib/systemd/sysext.conf</filename></member>
+      <member><filename>/usr/lib/systemd/sysext.conf</filename></member>
+    </simplelist></para>
```

Comment:
I don't think there's any reason to *not* parse drop-ins. It is generally useful because it allows configuration to composed. So please just use the full-featured function that also takes dropins and extend the list here.

Reply:
This would even match with the config parsing functions that are currently exported :)